homeassistant/components/husqvarna_automower_ble/config_flow.py

```diff
@@ -34,12 +34,8 @@ def _is_supported(discovery_info: BluetoothServiceInfo):
         service == "98bd0001-0b0e-421a-84e5-ddbf75dc6de4"
```

Comment:
This is the generic scan service that is common to both mowers and all other bluetooth devices from gardena: https://github.com/elupus/gardena-bluetooth/blob/61ff7d111a27cccf6ae608ea7684e75d906fee30/gardena_bluetooth/const.py#L24

This need to filter on the manufacturer data and look at device types:

https://github.com/elupus/gardena-bluetooth/blob/61ff7d111a27cccf6ae608ea7684e75d906fee30/gardena_bluetooth/parse.py#L22